test/unit/org/apache/cassandra/config/DatabaseDescriptorTest.java

```diff
@@ -590,4 +590,23 @@ public void testApplyTokensConfigInitialTokensOneNumTokensNotSet()
         Assert.assertEquals(Integer.valueOf(1), config.num_tokens);
         Assert.assertEquals(1, DatabaseDescriptor.tokensFromString(config.initial_token).size());
     }
+
+    @Test (expected = ConfigurationException.class)
+    public void testInvalidSub1DefaultRFs() throws ConfigurationException
+    {
+        DatabaseDescriptor.setDefaultKeyspaceRF(0);
+    }
+
+    @Test (expected = ConfigurationException.class)
+    public void testInvalidSub0MinimumRFs() throws ConfigurationException
+    {
+        DatabaseDescriptor.setMinimumKeyspaceRF(-1);
+    }
+
+    @Test (expected = ConfigurationException.class)
+    public void testDefaultRfLessThanMinRF()
```

Comment:
I think it makes sense to add one more test where we set default at first and then minimum and it triggers the exception. I think this condition is not covered at the moment.

Reply:
This one also does not seem to be covered:
```
if (conf.default_keyspace_rf < conf.minimum_keyspace_rf)
```